btree.go

```diff
@@ -157,6 +159,16 @@ func (btree *btreeTable) decodeRecord(payload []byte) (Record, error) {
 			recbuf = recbuf[int(n):]
 			v = vv
 
+		case StInt24:
+			bs := make([]byte, 4)
+			if n := copy(bs[1:], recbuf); n != 3 {
+				panic(fmt.Sprintf("read %d bytes", n))
+			}
+			if bs[1]&0x80 > 0 {
+				bs[0] = 0xff
+			}
+			v = binary.BigEndian.Uint32(bs)
```

Comment:
I need to still drop the sign bit from `bs[1]`, right?

Reply:
Nope. Should be correct now.